src/components/global/Playground/index.tsx

```diff
@@ -13,18 +13,39 @@ import TabItem from '@theme/TabItem';
 
 import { IconHtml, IconTs, IconVue, IconDefault, IconCss, IconDots } from './icons';
 
-const ControlButton = ({ isSelected, handleClick, title, label }) => {
-  return (
+const ControlButton = ({
+  isSelected,
+  handleClick,
+  title,
+  label,
+  disabled,
+}: {
+  isSelected: boolean;
+  handleClick: () => void;
+  title: string;
+  label: string;
+  disabled?: boolean;
+}) => {
+  const controlButton = (
     <button
-      type="button"
```

Comment:
Did you mean to remove the `type="button"` here?

Reply:
Yes. When not inside of a `<form>` element, there is no need to specify the type. I was being verbose in the original implementation and forgot to clean this up.